pkg/sfu/codecmunger/codecmunger.go

```diff
@@ -0,0 +1,25 @@
+package codecmunger
```

Comment:
Introducing an interface which can be used to implement any codec specific munger. Currently, only VP8.

Not sure if we will need some form of munging when simulcast support is added for AV1/VP9. Actually, I guess we can do VP9 right now. That is something I will try after this PR settles.